zyte_common_items/pages.py

```diff
@@ -54,6 +61,32 @@ def _get_metadata_class(cls: type) -> Optional[Type[MetadataT]]:
     return get_generic_param(cls, HasMetadata)
 
 
+class PriceMixin(FieldsMixin):
+    """Provides price-related field implementations."""
+
+    _parsed_price: Optional[Price] = None
+
+    async def _get_parsed_price(self):
+        if not hasattr(self, "price"):
+            return None
```

Comment:
Is it the check which makes us call user's price field multiple times? If self.parsed_price is not None, we don't need to do this check anymore, right? 

And if it is None, would it work if you e.g. assing it to a varibale, or use getattr, to avoid accessing self.price two times here?

Reply:
I was able to reduce the calls but if you access `currencyRaw` and then access `price` it's still called twice, as the result of the first call isn't saved anywhere. I don't think we can do anything about it?